sensu.gemspec

```diff
@@ -16,7 +16,11 @@ Gem::Specification.new do |s|
   s.add_dependency "json" if RUBY_VERSION < "1.9"
   s.add_dependency "multi_json", "1.11.0"
   s.add_dependency "uuidtools", "2.1.4"
-  s.add_dependency "eventmachine", "1.0.3"
+  if RUBY_PLATFORM =~ /mingw/
```

Comment:
This cannot work when publishing the gem to rubygems, as it will use the platform it is pushed from.